lib/internal/crypto/webcrypto.js

```diff
@@ -238,7 +251,7 @@ async function deriveKey(
       throw lazyDOMException('Unrecognized name.');
   }
 
-  return importKey('raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
+  return importKey.call(this, 'raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
```

Comment:
This is relying on the globally mutable `%Function.prototype%.call`
```suggestion
  return ReflectApply(importKey, this, ['raw', bits, derivedKeyAlgorithm, extractable, keyUsages]);
```
```suggestion
  return FunctionPrototypeCall(importKey, this, 'raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
```

or more simply (I don't think it matters what value of `this` we pass)
```suggestion
  return importKey('raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
```

Reply:
of course it does, this PR adds `this` check to all the methods. I'll use ReflectApply.